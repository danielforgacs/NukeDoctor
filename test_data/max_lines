Node {
    line 2
    line 3
    line 4
    line 5
    line 6
    line 7
}
Node {
    line 2
    line 3
    line 4
}
Node {
    line 2
    line 3
    line 4
    line 5
    line 6
    line 7
}
Node {
    line 2
}
Node {
    line 2
    line 3
    line 4
    line 5
    line 6
    line 7
}