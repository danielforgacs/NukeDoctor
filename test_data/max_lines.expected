Node {
    line 2
    line 3
    line 4
}
Node {
    line 2
}
